dockerfiles/theia/Dockerfile

```diff
@@ -23,16 +23,24 @@ WORKDIR /home/theia
 # have default Theia build in the workspace in case no plugins are requested
 ADD https://raw.githubusercontent.com/theia-ide/theia/v${THEIA_VERSION}/examples/browser/package.json /home/theia/package.json
 ADD theia-default-package.json /home/default/theia/package.json
-ADD versions.sh /home/default/versions.sh
+ADD src/versions.sh /home/default/versions.sh
```

Comment:
maybe not for this PR, but should we store in a directory all files that will go to /home/default so we can use only one ADD instruction, and we don't need to think on adding line each time we require a file ?